### PR TITLE
8296675: Exclude linux-aarch64 in NSS tests

### PR DESCRIPTION
**Pull request by wangweij**
The NSS tests do not work fine on linux-aarch64 now. Disable them at the moment.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8296675](https://bugs.openjdk.org/browse/JDK-8296675): Exclude linux-aarch64 in NSS tests

### Reviewers
 * [Daniel D. Daugherty](https://openjdk.org/census#dcubed) (@dcubed-ojdk - **Reviewer**) ⚠️ Review applies to [4126608e](https://git.openjdk.org/jdk/pull/11063/files/4126608eb32776961ff8f16f99bdd9652ab49cdc)
 * [Sean Mullan](https://openjdk.org/census#mullan) (@seanjmullan - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/11063/head:pull/11063` \
`$ git checkout pull/11063`

Update a local copy of the PR: \
`$ git checkout pull/11063` \
`$ git pull https://git.openjdk.org/jdk pull/11063/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 11063`

View PR using the GUI difftool: \
`$ git pr show -t 11063`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/11063.diff">https://git.openjdk.org/jdk/pull/11063.diff</a>

</details>

### CODE REVIEW

**Review by dcubed-ojdk — APPROVED** [review]
Thumbs up. This is a trivial fix.

Do we know if these tests will also fail on linux-x64 OL9?

**Comment by seanjmullan** [comment]
I guess it's not possible to ProblemList an OS/arch combo?

File: test/jdk/sun/security/pkcs11/PKCS11Test.java [hunk 1]
```diff
@@ -700,10 +700,10 @@ private static Map<String, String[]> getOsMap() {
         osMap.put("Linux-arm-32", new String[] {
                 "/usr/lib/arm-linux-gnueabi/nss/",
                 "/usr/lib/arm-linux-gnueabihf/nss/" });
-        osMap.put("Linux-aarch64-64", new String[] {
-                "/usr/lib/aarch64-linux-gnu/",
-                "/usr/lib/aarch64-linux-gnu/nss/",
-                "/usr/lib64/" });
+//        osMap.put("Linux-aarch64-64", new String[] {
```
Comment (seanjmullan) [inline test/jdk/sun/security/pkcs11/PKCS11Test.java:703]: Perhaps add a comment above this noting why these are being excluded. Could include the bugid for now until it is fixed.

**Comment by seanjmullan** [comment]
> > I guess it's not possible to ProblemList an OS/arch combo?
> 
> Yes, we can, but there are quite a lot of tests failing here.

Seems fine then.

**Review by seanjmullan — APPROVED** [review]
Bug needs `noreg-self` label. Otherwise, looks good.

**Pull request closed without merging**